### DISCUSSION TITLE
Ouverture cartouche

### DISCUSSION THREAD

**Discussion by Ptigege** [cgeo/WhereYouGo#437] [Ideas]
Bonjour est il possible d'autoriser l'application à voir tous les fichiers du téléphone. Cela permettrai de corriger une erreur. Car quand on veut sélection le fichier d'ouverture de la cartouche on ne peut pas se lectionner les dossiers qu'on veut.

Est il possible également que l'application soit proposé dans ouvrir avec lorsque qu'on veut ouvrir un fichier gwc ?

**Comment by FriedrichFroebel** [comment]
Judging from the automatic translation of your comment, the first part seems to be the same as https://github.com/cgeo/WhereYouGo/issues/309, 

<details>
<summary>Automatic translation of the original comment (just for the record)</summary>

> Hello is it possible to authorize the application to see all the files on the phone. This will correct an error. Because when you want to select the file of the cartridge to open, you cannot select the files you want.
> 
> Is it also possible that the application is offered in open with when you want to open a gwc file?
</details>
